app/javascript/components/visualization/controls/ScatterPlotLegend.js

```diff
@@ -102,6 +111,20 @@ function LegendEntry({
   )
 }
 
+/** Handle click on "Show all" or "Hide all" button */
+function showHideAll(showOrHide, labels, updateShownTraces) {
```

Comment:
I'm seeing double events for the show/hide all. One is these events that are added here and the other is a `click:link` event. Not sure it's a problem but just wanted to call it out that it might be adding noise to have both?

Reply:
concur.  the links already have a data-analytics-name on them, so I don't think the extra property logging is worth the extra complexity

Reply:
Good catch!  I've eliminated the double logging (94522ee).